### ISSUE TITLE
Map of other patrons to show en route locations

### ISSUE THREAD

**Issue by matkastner** [pub-thursday/pub-thursday-android#10]

**Comment by dylan8902** [comment]
I'll get straight on this

**Comment by dylan8902** [comment]
actually, en-routing may be more important before there is a map

**Comment by matkastner** [comment]
Yeah that's true, got a pretty big update coming in a sec. Added a Leaderboard class and separate request classes.

Was thinking the en route functionality should be possible in the dialog after pressing check in.

**Comment by dylan8902** [comment]
Ye makes sense to be en-routing until your location is valid
